templates/html/index.html

```diff
@@ -0,0 +1,101 @@
+                    <img src="../images/avatar.png" alt="Sign Up image">
+                </div>
+            </div>
+
+            <div class="board two">
+                <div class="board-header">
+                    <h2 class="board-number">2</h2>
+                    <h2 class="board-title">Create an announcement</h2>
+                </div>
+                <div class="board-body">
+                    <div class="board-description">
+                        <p>Lorem ipsum dolor sit amet consectetur adipiscing elit est vehicula torquent, consequat ridiculus sodales porta eleifend nam fermentum aenean diam urna, sociosqu ac pulvinar primis at libero accumsan curabitur lacus.</p>
+                    </div>
+                    <img src="../images/announcement.png" alt="announcement image">
+                </div>
+            </div>
+            
+            <div class="board three">
+                <div class="board-header">
+                    <h2 class="board-number">3</h2>
+                    <h2 class="board-title">View broadcaster's feedback</h2 c>
+                </div>
+                <div class="board-body">
+                    <div class="board-description">
+                        <p>Lorem ipsum dolor sit amet consectetur adipiscing elit est vehicula torquent, consequat ridiculus sodales porta eleifend nam fermentum aenean diam urna, sociosqu ac pulvinar primis at libero accumsan curabitur lacus.</p>
+                    </div>
+                    <img src="../images/feedback.png" alt="feedback image">
+                </div>
+            </div>
+        </section>
+    </div>
+
+    <section class="more-features">
+        <div class="update-feature ft">
+            <h3>Update an announcement anytime</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipiscing elit lacinia scelerisque, feugiat mus morbi at facilisis justo metus velit, cum potenti dis neque ad himenaeos platea parturient. Taciti eu euismod at habitasse fames nullam, in mi feugiat ad sed vivamus aliquam, sociosqu faucibus eget ornare purus. </p>
+        </div>
+    
+        <div class="view-feature ft">
+            <h3>View all or a specific announcement</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipiscing elit lacinia scelerisque, feugiat mus morbi at facilisis justo metus velit, cum potenti dis neque ad himenaeos platea parturient. Taciti eu euismod at habitasse fames nullam, in mi feugiat ad sed vivamus aliquam, sociosqu faucibus eget ornare purus. </p>
+        </div>
+    
+        <div class="announcement-state-feature ft">
+            <h3>View announcement of a specific state</h3>
+            <p>Lorem ipsum dolor sit amet consectetur adipiscing elit lacinia scelerisque, feugiat mus morbi at facilisis justo metus velit, cum potenti dis neque ad himenaeos platea parturient. Taciti eu euismod at habitasse fames nullam, in mi feugiat ad sed vivamus aliquam, sociosqu faucibus eget ornare purus. </p>
```

Comment:
Would you mind using real statement, that explains why `View announcement of a specific state` is useful. this will give more insight to the user since what you are working on is not just a template but a project UI

Reply:
I am definitely going to work on it. I will notify you once it's done.